src/components/Card.tsx

```diff
@@ -64,9 +65,16 @@ const appStyles = classnames(
   display('flex'),
   flexDirection('flex-col'),
   alignItems('items-stretch'),
-  minHeight('sm:min-h-card', 'min-h-fit')
+  minHeight('sm:min-h-card', 'min-h-fit'),
+  maxHeight('max-h-app-card'),
+  height('h-app-card'),
+  width('w-full')
```

Comment:
Looks like the container grows when the content changes, that shouldn't happen:

https://user-images.githubusercontent.com/49658988/186421918-cfd5cc6d-7082-47e7-a252-bbb3c7c1d58e.mp4

Reply:
Almost fixed it, container still grows a little, we have too many `w-full`, that's bad

Reply:
I finally fixed this, just added a maxWidth on the card container and let the cards take the full width

Reply:
Unfortunately I haven't been able to figure out why this is - will this block launch? If not I can log it in a separate trello ticket